x/wasm/module.go

```diff
@@ -280,7 +278,18 @@ func getExpectedLibwasmVersion() string {
 	return ""
 }
 
-func checkLibwasmVersion(cmd *cobra.Command, args []string) error {
+// CheckLibwasmVersion ensures that the libwasmvm version loaded at runtime matches the version
+// of the github.com/CosmWasm/wasmvm dependency in go.mod. This us useful when dealing with
+// shared libraries that are copied or moved from their default location, e.g. when building the node
+// on one machine and deploying it to other machines.
+//
+// Usually the libwasmvm version (the Rust project) and wasmvm version (the Go project) match. However,
+// there are situations in which this is not the case. This can be during development or if one of the
+// two is patched. In such cases it is advised to not execute the check.
+//
+// An alternative method to obtain the libwasmvm version loaded at runtime is executing
+// `wasmd query wasm libwasmvm-version`.
+func CheckLibwasmVersion(cmd *cobra.Command, args []string) error {
```

Comment:
Thanks for the comments. These are very valid scenarios that need to be considered!
The version check was a very helpful in the past and I would be sad to let it go.
Some options:
a) move the integration to the `cmd/wasmd/root.go` example CLI 
b) introduce a new start cmd flag `--wasm.skip_wasmvm_version_check` (or better name)
c) Add toggle param to `AddModuleInitFlags(startCmd *cobra.Command, skipVersionCheck bool)` which is called by `root.go`

I am in favour of option b) as it does not require code changes for different scenarios

Reply:
ok, then let's go with option b). For a long term solution, a project can always replace the call to the [default init flags](https://github.com/CosmWasm/wasmd/blob/v0.40.0-rc.0/cmd/wasmd/root.go#L173) with a custom solution. No need to optimize for that

Reply:
okay, cool. Could you take over this PR?